docs/resources/Channel.md

```diff
@@ -258,7 +260,7 @@ Represents a message sent in a channel within Discord.
 | Field       | Type      | Description                             |
 |-------------|-----------|-----------------------------------------|
 | message_id? | snowflake | id of the originating message           |
-| channel_id?  | snowflake | id of the originating message's channel |
+| channel_id  | snowflake | id of the originating message's channel |
```

Comment:
is this supposed to be marked as required again here?

Reply:
Yeah I mistakenly made it optional in one of the passes.

Reply:
JK I was wrong. It's optional when _sending_ a reply, but will always be included when you receive this structure in an event/endpoiint.